### ISSUE TITLE
Spammy gui warnings

### ISSUE THREAD

**Issue by btbest** [ilastik/ilastik#2886]
While testing 1.4.1b17 I noticed that we have various sources of random spam in the console window next to the GUI on Windows and Linux. On Mac there doesn't seem to be an extra console window (?), so not an issue. On Windows and Linux it's a bit irritating though.

###  QFont::fromString: Invalid description '(empty)'
On Windows, from at least 1.4.1b15 onwards. Spammed in the data selection applet when moving the horizontal separator bar between the input data table and the viewer below it. I'm also getting this printed once when switching back from another applet to data selection.

### QOpenGLFramebufferObject: Framebuffer incomplete attachment.
On Windows, possibly only introduced in 1.4.1b17. `QOpenGLFramebufferObject: Framebuffer incomplete attachment.` and `QOpenGLFramebufferObject: Framebuffer incomplete, missing attachment.` printed ~6 times when opening a project. I'm not seeing this in 1.4.1b15.

## Screenshots
![image](https://github.com/ilastik/ilastik/assets/63287233/7e4a8ccb-0f32-4ae5-b154-375142c2e6e5)

**Comment by k-dominik** [comment]
can't reproduce the

```
run_ilastik.sh: 6: [[: not found
run_ilastik.sh: 16: [[: not found
run_ilastik.sh: 25: [[: not found
run_ilastik.sh: 25: [[: not found
run_ilastik.sh: 32: [[: not found
run_ilastik.sh: 32: [[: not found
run_ilastik.sh: 44: [[: not found
run_ilastik.sh: 49: [[: not found
```

bit. I tried bash and zsh on my linux machine. Would you have a little more detail on how you got these?

**Comment by emilmelnikov** [comment]
> can't reproduce the
> 
> ```
> run_ilastik.sh: 6: [[: not found
> run_ilastik.sh: 16: [[: not found
> run_ilastik.sh: 25: [[: not found
> run_ilastik.sh: 25: [[: not found
> run_ilastik.sh: 32: [[: not found
> run_ilastik.sh: 32: [[: not found
> run_ilastik.sh: 44: [[: not found
> run_ilastik.sh: 49: [[: not found
> ```
> 
> bit. I tried bash and zsh on my linux machine. Would you have a little more detail on how you got these?

This is probably coming from `dash` or a similar POSIX-compatible shell that doesn't support ["bashisms"](https://mywiki.wooledge.org/Bashism).

**Comment by Tomaz-Vieira** [comment]
Weird. The `start_ilastik.sh` script has the shebang pointing to bash, though... How are you running the script?

**Comment by btbest** [comment]
Might be an issue specific to [konsole](https://konsole.kde.org/) then

**Comment by Tomaz-Vieira** [comment]
> Might be an issue specific to [konsole](https://konsole.kde.org/) then

I don't think so. Konsole is just a terminal emulator, and shouldn't care what shell you use

**Comment by btbest** [comment]
Got a new one:

```
WARNING 2024-10-17 10:08:47,348 logger 12216 11884 __init__.py:(91): UserWarning: UserWarning('Valid PROJ data directory not found. Either set the path using the environmental variable PROJ_DATA (PROJ 9.1+) | PROJ_LIB (PROJ<9.1) or with `pyproj.datadir.set_data_dir`.')
```

Hopefully we can "just" loglevel suppress this one

**Comment by k-dominik** [comment]
could you maybe cross out the `QFont::fromString: Invalid description '(empty)'` in your original post? That one is gone.

**Comment by btbest** [comment]
Seem to all be fixed as of 1.4.1rc2
